src/test/java/org/jabref/logic/citationkeypattern/BracketedPatternTest.java

```diff
@@ -380,4 +380,51 @@ void expandBracketsWithTestCasesFromRegExpBasedFileFinder() {
                 "Eric von Hippel and Georg von Krogh have published Open Source Software and the \"Private Collective\" Innovation Model: Issues for Organization Science in Organization Science.",
                 BracketedPattern.expandBrackets("[author] have published [title] in [journal].", ',', entry, database));
     }
+
+    @Test
+    void expandBracketsWithFallBackToString() {
+        BibEntry bibEntry = new BibEntry()
+                .withField(StandardField.TITLE, "Title");
+
+        assertEquals("Title_eprint", BracketedPattern.expandBrackets("[title]_[EPRINT:(eprint)]", null, bibEntry, null));
+    }
+
+    @Test
+    void expandBracketsWithTWOFallBack() {
+        BibEntry bibEntry = new BibEntry()
+                .withField(StandardField.YEAR, "2021");
+
+        assertEquals("2021", BracketedPattern.expandBrackets("[title:([EPRINT:([YEAR])])]", null, bibEntry, null));
+    }
+
+    @Test
+    void unbalancedBracketsExpandBracketsWithFallBack() {
```

Comment:
This doesn't follow the [naming convention](https://devdocs.jabref.org/getting-into-the-code/code-howtos#test-cases) `methodUnderTestExpectedBehaviorContext`